public/js/components/Editor.js

```diff
@@ -266,9 +265,12 @@ const Editor = React.createClass({
       enableCodeFolding: false,
       gutters: ["breakpoints"],
       value: " ",
-      extraKeys
+      extraKeys: {}
     });
 
+    // disables the default search shortcuts
+    this.editor._initShortcuts = () => {};
```

Comment:
I'm a little sad that we're diverging on another thing. It would be nice to work on the official editor in m-c instead of customizing it to our needs, but I know that's difficult right now because we need to separate it out into a module for that to really happen.

It's OK for now.